### ISSUE TITLE
Define symbol options in comment of multimap

### ISSUE THREAD

**Issue by sts** [rspamd/rspamd#4468]
## Summary

Allow to include match details when using the multimap module to eg. filter headers by regex regex. Eg. sample map:

```
/Neighbou?r/i  BAD_SUBJECT:6 # rule:001, regex:neighbour
```

Expected output:

```
BAD_SUBJECT(6.00){rule:001, regex:neighbour}
```

## Motivation

Allow identifying the exact matched rule when reviewing a message's spam report.

## Describe alternatives you've considered

Numbering the rule in the symbol name has the downside that each symbol must be defined in the multimap definiton. Eg.:

```
/Neighbou?r/i  BAD_SUBJECT_001:6
```

```
multimap {
  "bad_headers {
    type = content;
    filter = headers;
    map = /etc/rspamd/maps.d/bad_headers.map;
    score = 1.0;
    regexp = true;
    multi = true;
    symbols = [ "BAD_SUBJECT_001", .... ]
  }
}
```

**Comment by vstakhov** [comment]
No, I don't like the idea to use comments for that purposes. We can probably introduce syntax like `<symbol_name>[:<symbol_score[:[option1,...optionN]]` (e.g. `SYMBOL:1.0:opt1,opt2).

**Comment by sts** [comment]
Tested & works, thanks a lot!

One thing I noted, is that the [option will only return](https://github.com/rspamd/rspamd/commit/0b7f9101e38473b5c42d068aea884a527ec506b6#diff-502b799106b44508c200e93c90c818d1f14472bf0836d1b418f946be5ec0c2b7R524) if it validates the symbol format & starts with `[a-zA-Z]`. Not sure if thats intended.

Also the [comment](https://github.com/rspamd/rspamd/blob/0b7f9101e38473b5c42d068aea884a527ec506b6/src/plugins/lua/multimap.lua#L495) above the `parse_ret` function might need an update.
